code/datums/components/anti_magic.dm

```diff
@@ -135,8 +142,8 @@
 	// Block success! Add this parent to the list of antimagic sources
 	antimagic_sources += parent
 
+	drain_antimagic?.Invoke(source, parent)
 	if((charges != INFINITY) && charge_cost > 0)
-		drain_antimagic?.Invoke(source, parent)
```

Comment:
`drain_antimagic`:

`invoked when we have been drained a antimagic charge`

But you've changed it so it's invoked when any blocking happens period - 

Which is fine, but you should update the documentation (and probably the var name)